src/test/datascience/uiTests/ipywidget.ui.functional.test.ts

```diff
@@ -220,20 +220,19 @@ use(chaiAsPromised);
                 assert.include(outputHtml, '<input type="checkbox');
             });
         });
-        test('Render ipysheets', async () => {
+        test('Render ipysheets', async (done) => {
             const { notebookUI } = await openIPySheetsIpynb();
             await assert.eventually.isFalse(notebookUI.cellHasOutput(3));
 
-            await notebookUI.executeCell(1);
-            await notebookUI.executeCell(3);
-
             await retryIfFail(async () => {
+                await notebookUI.executeCell(1);
+                await notebookUI.executeCell(3);
                 const cellOutput = await notebookUI.getCellOutputHTML(3);
 
                 // Confirm cells with output has been rendered.
                 assert.include(cellOutput, 'Hello</td>');
                 assert.include(cellOutput, 'World</td>');
-            });
+            }).catch(done);
```

Comment:
This won't make any difference.

`done` is meant to be used without the `async` keyword.
Old style async code so we can tell the calling code via a callback when the function is done.

Reply:
Oh okay. I thought that was what this error needed to be resolved. I guess it legitimately timed out then.
> Timeout of 180000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

Reply:
Yup, legitimately timedout.

Reply:
You'll need to increase the time out of the tests.
However `180000ms` is 3 minutes, thats long enough for individual tests.

Reply:
Reverted this change. I haven't fixed the ipywidgets tests completely but this branch still contains changes that I think are necessary, so going to merge unless anyone objects